ngraph_creator/operations/src/OperationsBase.cpp

```diff
@@ -77,6 +78,16 @@ void OperationsBase::setNgraphNodes(std::shared_ptr<NgraphNodes> nodes) { mNgrap
 
 bool OperationsBase::validate() { return true; }
 
+bool OperationsBase::validateForPlugin() {
```

Comment:
@akodanka validateForPlugin is a confusing name for a function that checks for valid input dimensions. I am assuming this check is to skip CTS/VTS tests where input dimensions are provided only at runtime? Can we rename to something like checkInputDimensions??

Reply:
The idea for validate API is not only to validate for input dimensions, but rather handle any unsupported conditions that might arise for ngraph API usage.
And the reason for using validateForPlugin is to align with the naming of createNodeForPlugin. These 2 APIs can be override to handle any deviations in the plugin specific operation implementation.

Reply:
I would suggest to keep the validation code to validate function.

Reply:
This specific common handling for input index 0 for each operation, helps fix the CTS crashes.
I agree this validation needs to be done at each operation, as the inputs to each operation is different. 
As discussed, since this requires collaborative effort, this will be completed as part of a separate task(PR) soon.